examples/tests/sdk_tests/std/string.w

```diff
@@ -39,19 +39,28 @@ test "length" {
 //-----------------------------------------------------------------------------
 // at()
 
+let INDEX_OUT_OF_BOUNDS_ERROR = "index out of bounds";
+
 assert("boom".at(2) == "o");
 // Negative integers count back from the last string character.
 assert("boom".at(-4) == "b");
 assert("boom".at(-1) == "m");
+// Should throw an exception
+assertThrows(INDEX_OUT_OF_BOUNDS_ERROR, () => {
+  "boom".at(4);
+});
+assertThrows(INDEX_OUT_OF_BOUNDS_ERROR, () => {
+  "boom".at(-5);
+});
 
 test "at()" {
   assert("boom".at(0) == "b");
   // Negative integers count back from the last string character.
   assert("boom".at(-4) == "b");
   assert("boom".at(-1) == "m");
-  
-  // Should throw an exception instead of returns undefined see issue #3240
-  // assert("boom".at(-10) == "m");
+  // Should throw an exception
+  try { "boom".at(4); } catch s { assert(s == INDEX_OUT_OF_BOUNDS_ERROR ); }
+  try { "boom".at(-5); } catch s { assert(s == INDEX_OUT_OF_BOUNDS_ERROR ); }
```

Comment:
Is there a reason it's not `assertThrows`? 
since we are visiting the `catch` scope only if there is an error, it will pass either way...
It seems like it can be removed :)

Reply:
True, but trying to call `assertThrows` in `test {}` throws `error: Cannot call into preflight phase while inflight`. I think that's the reason why many tests doing this that way

Reply:
So sure it can be removed, but then we should remove it in many tests, I prefer to be consistent 😅

Reply:
> True, but trying to call `assertThrows` in `test {}` throws `error: Cannot call into preflight phase while inflight`. I think that's the reason why many tests doing this that way

That's a good point!! I didn't know that, can you please open an issue and add a comment next to those lines? (no need to remove them), it would be super helpful 🙏

Reply:
okidoki 🙏✅